tcstore-exporter/src/main/java/org/terracotta/store/export/ParquetExportStats.java

```diff
@@ -132,4 +135,26 @@ public boolean getExportSuccess() {
     protected void setExportSuccess(boolean exportStatus) {
         this.exportSuccess = exportStatus;
     }
+
+    /**
+     * @return the name(s) of the generated parquet file(s).
+     */
+    public List<String> getFilenames() {
+        return filenames;
+    }
+
+    public void setFilenames(List<String> filenames) {
```

Comment:
setter could/should be protected rather than public.

Reply:
done